modules/addDate.js

```diff
@@ -0,0 +1,19 @@
+import { dateElement } from './selector.js';
+
+import DateTime from '../node_modules/luxon/src/datetime.js';
+
+const now = DateTime.now();
+const addDate = () => {
+  const formatted = now.toLocaleString({
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    hour12: true,
+  });
+  dateElement.innerHTML = formatted;
+};
+
+export default addDate;
```

Comment:
- [ ] Good work here, champ! To further improve the quality of your user experience, I think it will be nice to have the time update in realtime. You can use a `setTimeInterval()` callback method for this to call the function every `1 sec` /`1000ms` 💯

Reply:
Great job on implementing this change :+1: